BigAssApi.js

```diff
@@ -60,6 +60,26 @@ function FanMaster (numberOfExpectedFans) {
             this.rescanForFans();
         }
     }.bind(this)
+    */
+        handleNewFan = function(msg, address) {
```

Comment:
Indentation - also I'd rather you just remove+replace the code you commented out